api/metrics/permissions.py

```diff
@@ -14,6 +16,14 @@ class IsInstitutionalMetricsUser(permissions.BasePermission):
 
     def has_object_permission(self, request, view, obj):
         auth = get_user_auth(request)
-        if auth.user.has_perm('view_institutional_metrics', obj.institution):
+        if type(obj) != Institution:
+            obj = obj.institution
+        if not auth.user:
+            return False
+        if auth.user.has_perm('view_institutional_metrics', obj):
             return True
         return False
+
+    def has_permission(self, request, view):
```

Comment:
This just feels wrong to override, I can find a lot of counter examples where it doesn't seem to be necessary to override this, but I not sure how they are actually working. I'd leave this in, because it should work anyway.